docs/guides/staking/overview.md

```diff
@@ -0,0 +1,213 @@
+    validators can be slashed after an infraction
+* Atom provisions - The process of increasing the Atom supply.  Atoms are
+    periodically created on the Cosmos Hub and issued to bonded Atom holders.
+    The goal of inflation is to incentize most of the Atoms in existence to be
+    bonded. Atoms are distributed unbonded and using the fee_distribution mechanism
+* Transaction fees - transaction fee is a fee that is included in a Cosmsos Hub
+    transaction. The fees are collected by the current validator set and 
+    distributed among validators and delegators in proportion to their bonded 
+    Atom share
+* Commission fee - a fee taken from the transaction fees by a validator for 
+    their service 
+
+## The pool and the share
+
+At the core of the Staking module is the concept of a pool which denotes a
+collection of Atoms contributed by different Atom holders. There are three
+pools in the Staking module: the bonded, unbonding, and unbonded pool.  Bonded
+Atoms are part of the global bonded pool. If a validator or delegator wants to
+unbond its Shares, these Shares are moved to the the unbonding pool for the
+duration of the unbonding period. From here normally Atoms will be moved
+directly into the delegators wallet, however under the situation thatn an
+entire validator gets unbonded, the Atoms of the delegations will remain with
+the validator and moved to the unbonded pool.  For each pool, the total amount
+of bonded, unbonding, or unbonded Atoms are tracked as well as the current
+amount of issued pool-shares, the specific holdings of these shares by
+validators are tracked in protocol by the validator object. 
+
+A share is a unit of Atom distribution and the value of the share
+(share-to-atom exchange rate) can change during system execution. The
+share-to-atom exchange rate can be computed as:
+
+`share-to-atom-exchange-rate = size of the pool / ammount of issued shares`
+
+Then for each validator (in a per validator data structure) the protocol keeps
+track of the amount of shares the validator owns in a pool. At any point in
+time, the exact amount of Atoms a validator has in the pool can be computed as
+the number of shares it owns multiplied with the current share-to-atom exchange
+rate:
+
+`validator-coins = validator.Shares * share-to-atom-exchange-rate`
+
+The benefit of such accounting of the pool resources is the fact that a
+modification to the pool from bonding/unbonding/slashing of Atoms affects only
+global data (size of the pool and the number of shares) and not the related
+validator data structure, i.e., the data structure of other validators do not
+need to be modified. This has the advantage that modifying global data is much
+cheaper computationally than modifying data of every validator. Let's explain
+this further with several small examples: 
+
+XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
```

Comment:
Not reviewing this example for now, agree it could be clarified and simplified.

Reply:
yeah not top priority